### PR TITLE
openThread in docs

### PR DESCRIPTION
**Pull request by RachBLondon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ console.log(profile)
...
 
 #### 1. Create a 3Box instance
-To create a 3Box session you call the `create` method. This creates an instance of the Box class which can be used to joinThreads and authenticate the user in any order. In order to create a 3Box session a `provider` needs to be passed. This can be an `ethereum provider` (from `web3.currentProvider`, or `window.ethereum`) or a `3ID Provider` (from [IdentityWallet](https://github.com/3box/identity-wallet-js)).
+To create a 3Box session you call the `create` method. This creates an instance of the Box class which can be used to joinThreads, openThreads and authenticate the user in any order. In order to create a 3Box session a `provider` needs to be passed. This can be an `ethereum provider` (from `web3.currentProvider`, or `window.ethereum`) or a `3ID Provider` (from [IdentityWallet](https://github.com/3box/identity-wallet-js)).
```
Comment (oed) [inline README.md]: nice catch!

Comment (oed) [reply README.md]: Actually maybe we should just remove `joinThreads`?

Comment (RachBLondon) [reply README.md]: It is still possible to joinThreads though? I think if its not harming maybe its good to leave in?

Comment (oed) [reply README.md]: yeah, but you can only call `joinThread` on a space, i.e. it's not possible to call before authenticating.

**Comment by RachBLondon** [comment]
@oed the CircleCI tests have failed, but is only changes to the markdown files - is it OK to merge?

**Comment by oed** [comment]
Yep, merging!

**Pull request merged**